### ISSUE TITLE
Nice tools! Thanks!

### ISSUE THREAD

**Issue by Snawoot** [inflex/ideviceunback#6]
Nice tools you have here! Thank you!

**Comment by pabs3** [comment]
Indeed, it is very helpful to workaround a few libimobiledevice issues:

https://github.com/libimobiledevice/libimobiledevice/issues/1147
https://github.com/libimobiledevice/libimobiledevice/issues/1005
https://github.com/libimobiledevice/libimobiledevice/issues/528
